cg/cli/upload.py

```diff
@@ -147,9 +149,20 @@ def delivery_report(context, family_id, print_console):
                                                                       tb_api.get_family_root_dir(
                                                                         family_id))
         hk_api = context.obj['housekeeper_api']
-        result = _add_delivery_report_to_hk(delivery_report_file, hk_api, family_id)
-        if result:
+        added_file = _add_delivery_report_to_hk(delivery_report_file, hk_api, family_id)
+
+        if added_file:
+            click.echo(click.style('uploaded to housekeeper', fg='green'))
+            _add_delivery_report_to_scout(context, added_file.full_path, family_id)
+            click.echo(click.style('uploaded to scout', fg='green'))
             _update_delivery_report_date(status_api, family_id)
+        else:
+            click.echo(click.style('already uploaded to housekeeper, skipping'))
+
+
+def _add_delivery_report_to_scout(context, path, case_id):
+    scout_api = scoutapi.ScoutAPI(context.obj)
+    scout_api.upload_delivery_report(path, case_id, update=True)
```

Comment:
Is there a way we can get somesort of status of success returned? e.g. True when the upload succeeds and False when it fails?

Reply:
fixed